generators/app/templates/src/styles/base/_variables.scss

```diff
@@ -19,12 +19,13 @@ $colors: (
 
 //---------------------------------------------------------------
 // Easing
+// Moved to util/_eases and can be accessed with a function
```

Comment:
yeah, I mostly put it in there because I'm not sure how closely people follow these threads.